diffrax/_brownian/path.py

```diff
@@ -138,15 +141,24 @@ def _evaluate_leaf(
         t1: RealScalarLike,
         key,
         shape: jax.ShapeDtypeStruct,
-        levy_area: type[Union[BrownianIncrement, SpaceTimeLevyArea]],
+        levy_area: type[
+            Union[BrownianIncrement, SpaceTimeLevyArea, SpaceTimeTimeLevyArea]
+        ],
         use_levy: bool,
     ):
         w_std = jnp.sqrt(t1 - t0).astype(shape.dtype)
+        key_w, key_hh, key_kk = jr.split(key, 3)
```

Comment:
Can we do the split inside the `if` blocks, to avoid the overhead in the common case of no Levy area?

Reply:
I wanted to keep the generated path the same regardless of the Levy area setting, but fair, I guess this isn't really relevant for UBP.

Reply:
And I think I was using the wrong key to generate the bm anyway, oops.